fetch-utils.js

```diff
@@ -1,12 +1,28 @@
-const SUPABASE_URL = '';
-const SUPABASE_KEY = '';
+const SUPABASE_URL = 'https://swgiucfleoaonawjlvme.supabase.co';
+const SUPABASE_KEY = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InN3Z2l1Y2ZsZW9hb25hd2psdm1lIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NDQ0NTIxNjEsImV4cCI6MTk2MDAyODE2MX0.KwPiwmAFu_q_7yxOSUYv4k6Ejgu8uDsvfsGLixOpD9o';
 
 const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
 export function getUser() {
     return client.auth.session() && client.auth.session().user;
 }
+export async function getConcerts() {
 
+    const resp = await client.from('concert_title').select(`*, person_ticket(*)`);
+    return checkError(resp);
+}
+
+export async function createTicket(person) {
+    const resp = await client.from('person_ticket').insert(person);
+    console.log(person);
```

Comment:
just a little nudge to make sure you're getting rid of unnecessary console logs before pushing final code.